content/docs/accessibility.md

```diff
@@ -68,18 +78,20 @@ function Glossary(props) {
 }
 ```
 
-Use `<></>` syntax everywhere else:
+You can also use the [short syntax](/docs/fragments.html#short-syntax), if your tooling supports it, but only if you don't need any props, like the `key`:
```

Comment:
The point is great. Just some bulky language ([comma] if [comma] but only if).

Would suggest something like:
"When you don't need any props on the Fragment tag you can also use the [short syntax](/docs/fragments.html#short-syntax), if your tooling supports it."

It should already be clear from the example above that `key` is one of the props you could use on `Fragment`

Reply:
That's much better, thanks!